sbi/neural_nets/embedding_nets.py

```diff
@@ -31,3 +31,136 @@ def forward(self, x: Tensor) -> Tensor:
         """
         x = self.net(x)
         return x
+
+
+class CNNEmbedding(nn.Module):
+    def __init__(
+        self, input_dim: int, output_dim: int, num_fc: int = 2, num_hiddens: int = 120
+    ):
+        """multi-layer (C)NN
```

Comment:
Please start with capital letters (sorry for being picky, but these classes will be user-facing in the future)